src/data-workspace/inputs/use-min-max-limits.js

```diff
@@ -0,0 +1,16 @@
+import { useDataValueSet } from '../../shared/index.js'
+
+export const useMinMaxLimits = (dataElementId) => {
```

Comment:
I've ended up creating a small hook useMinMaxLimits which could potentially put to use in a refactor of the [<Limits />](https://github.com/dhis2/data-entry-app/blob/ac2831d459b60d3cb8e6bb71981176dcb23627be/src/data-workspace/data-details-sidebar/limits.js) component too.

Reply:
I don't think the refactor of `\<Limits \/\>` is a lot of work, I can do that now and we can see what it looks like - we can always drop the commit if we don't like it